staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/admission.go

```diff
@@ -297,6 +297,10 @@ func (a *MutatingWebhook) callAttrMutatingHook(ctx context.Context, h *v1beta1.W
 		return &webhookerrors.ErrCallingWebhook{WebhookName: h.Name, Reason: err}
 	}
 
+	for k, v := range response.Response.Annotations {
+		attr.SetAnnotations(h.Name, k, v)
```

Comment:
these names can theoretically overlap with the built in names. I guess we need `h.Name + ".mutatingwebhook"`.

Reply:
This h.Name is already full qualified, like `imagepolicy.kubernetes.io`. 
This name is set by cluster administrator. 
What about we trust cluster administrator that he will not set a name overlap with the built in names?

https://github.com/kubernetes/kubernetes/blob/f3942e7dedc0781aaee0a8dec1e134748b705ae2/staging/src/k8s.io/api/admissionregistration/v1beta1/types.go#L126

Reply:
Not sure it's obvious that this name shows up in auditing. But `imagepolicy.kubernetes.io.mutatingwebhook.admission.k8s.io` is not nice either.

Reply:
What about we ignore h.Name?
And allow remote admission to set pluginName themselves?
If so, remote admsssion will be consistent with the build-in admission.

Reply:
we could also enforce that webhooks use *.mutatingwebhook.admission.k8s.io as key.

Reply:
Done.